### PR TITLE
Added testimonials section

### PR DESCRIPTION
**Pull request by sarthakkundra**
Created a new folder src/views/HomeView/**Testimonials** having the card layout and an index.js file to map through the data in src/views/HomeViewData.js to show the testimonials. #28 

Side note :- I did a last minute change to make the alternate name dynamic in case there's no picture for the avatar which caused a merge conflict. Maybe this will help in resolving it. **File :-** src/views/HomeView/Testimonials/TestimonialCard.js **Line :-** 45

### CODE REVIEW

**Comment by Abhishek-kumar09** [comment]
Hey @sarthakkundra ,

CI is failing because you have many unused variables...Delete unused variables and deploy will work fine:)
image

**Comment by Abhishek-kumar09** [comment]
Also, rebase your branch to current development branch to resolve merge conflicts:
Its easy... just do
```
git remote add upstream https://github.com/codeforcauseorg/codeforcause.org.git
git fetch upstream
git rebase upstream/development
```
Then you will have to resolve merge conflicts on your own... There will be two code blocks one is written by you and other is written in current development branch... You can just delete any of the two codeblocks, that works for your PR.

once done:
run 
```
npm run start
```
to confirm everything is working fine.

Now do:
```
git push -f
```
and we can see your changes :1st_place_medal:

**Comment by sarthakkundra** [comment]
@Abhishek-kumar09 Thanks for the help will do the changes :)

**Comment by Abhishek-kumar09** [comment]
@sarthakkundra , Is it running in your local machine, the build is failing here.
Also, you have to rebase your PR anyways to get it merged with the latest code. .Please tell me when you stuck somewhere, we are there to help you.

**Comment by sarthakkundra** [comment]
@Abhishek-kumar09 I have made the changes and I was just pushing them after rebasing but I'm getting committing is not possible because I have unmerged files after resolving the conflict issues. I'm trying to fix that

**Comment by Abhishek-kumar09** [comment]
have you done rebasing?
you have to force push the branch after rebasing...because git history changes with rebasing.

**Comment by sarthakkundra** [comment]
@Abhishek-kumar09 Done! Thanks a lot for the help! 💯

**Comment by Abhishek-kumar09** [comment]
Booyah! you have made it...:)

**Comment by Abhishek-kumar09** [comment]
@GangaChatrvedi @KeenWarrior , please review.

File: src/views/pages/HomeView/Events.js [hunk 1]
```diff
@@ -38,10 +38,10 @@ const useStyles = makeStyles((theme) => ({
     height: '100%',
     display: 'flex',
     flexDirection: 'column',
-    position: 'relative',
+    position: 'relative'
   },
   cardMedia: {
-    paddingTop: '55.75%', // set 61.25% for 16:9 or highresolution images--- currently set to 55.75% to fit hqquality images.
+    paddingTop: '61.25%' // 16:9
```
Comment (Abhishek-kumar09) [inline src/views/pages/HomeView/Events.js:44]: Please undo this change.

**Comment by KeenWarrior** [comment]
Kudos to your work @sarthakkundra 

There are some changes.

1. Pull request created extra borders on top and bottom here.

Screenshot 2020-07-22 at 1 04 07 AM

2. HTML provided was for reference. You will need to use the design specified in the issue.
Screenshot 2020-07-22 at 1 05 40 AM

Please message here in case of any doubt in this PR. And on discord for any general concern.

**Comment by Abhishek-kumar09** [comment]
> There are some changes.
    Pull request created extra borders on top and bottom here.

This will be fixed by this change https://github.com/codeforcauseorg/codeforcause.org/pull/46#pullrequestreview-452765869

**Comment by sarthakkundra** [comment]
@KeenWarrior I think this PR should close this issue :)
Also @Abhishek-kumar09 once again thanks for all the help while making the PR!

**Comment by KeenWarrior** [comment]
Some more changes 😄 

- Please fix top and bottom padding according to [this](https://github.com/codeforcauseorg/codeforcause.org/blob/development/design/home_view/home_web.pdf)
- Hover on the picture should also create a shadow on the box.

**Comment by Abhishek-kumar09** [comment]
congrats @sarthakkundra , your first commit merged into codeforcause.org. Good Work!
